x-pack/test/functional_solution_sidenav/tests/search_sidenav.ts

```diff
@@ -65,6 +68,9 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
         // navigate back to the home page using header logo
         await solutionNavigation.clickLogo();
+        await solutionNavigation.sidenav.expectLinkActive({
+          deepLinkId: 'searchHomepage',
+        });
 
         // Redirected to Onboarding Page to Create Index
         await solutionNavigation.breadcrumbs.expectBreadcrumbExists({
```

Comment:
FYI I had to update this in my [nav update PR](https://github.com/elastic/kibana/pull/225709) since the global empty state won't have `Build` in it's breadcrumbs